docker-compose up elasticsearch fails.

### Info:
- Docker version (`Docker version 17.12.0-ce, build c97c6d6`): 
- Laradock commit (`0077de28a13feea669f9d46ef21c20477fa6dbb0`): 
- System info (Mac): 
- System info disto/version: 10.13.3

### Issue:
<!--- What seems to be wrong? -->
According to the documentation http://laradock.io/documentation/#use-elasticsearch one must run 
```
docker-compose up elasticsearch
```
To get ES running with (or by using Kibana ES also gets pulled). However, there seems to be no further documentation on what else todo. 

Im receiving the following error when starting up ES:

```
laradock 0077de2 ✗ 56d △ ➜ docker-compose up elasticsearch
Starting laradock_applications_1 ... done
Starting laradock_workspace_1 ... done
Starting laradock_php-fpm_1 ... done
Starting laradock_elasticsearch_1 ... done
Attaching to laradock_elasticsearch_1
elasticsearch_1        | [2018-03-07T08:28:50,443][WARN ][o.e.b.ElasticsearchUncaughtExceptionHandler] [] uncaught exception in thread [main]
elasticsearch_1        | org.elasticsearch.bootstrap.StartupException: java.lang.IllegalArgumentException: property [name] is missing in [/usr/share/elasticsearch/plugins/ingest-user-agent/plugin-descriptor.properties]
elasticsearch_1        | 	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:136) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:123) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:70) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:134) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.cli.Command.main(Command.java:90) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:91) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:84) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | Caused by: java.lang.IllegalArgumentException: property [name] is missing in [/usr/share/elasticsearch/plugins/ingest-user-agent/plugin-descriptor.properties]
elasticsearch_1        | 	at org.elasticsearch.plugins.PluginInfo.readFromProperties(PluginInfo.java:119) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.bootstrap.Spawner.spawnNativePluginControllers(Spawner.java:80) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:168) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:342) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:132) ~[elasticsearch-5.6.8.jar:5.6.8]
elasticsearch_1        | 	... 6 more
laradock_elasticsearch_1 exited with code 1
_____
```

So this means that either I forgot to set a setting somewhere that I'm not aware of or this is a bug on Laradock's side ? Googling this particular error didn't come up with any good results. 

### Expected behavior:
<!--- What should happen instead? -->
ES just starts as expected. 
_____

### Reproduce:
<!--- How can we reproduce the error? -->
Run the command `docker-compose up elasticsearch` on commit 0077de2.
_____

### Relevant Code:

```
NONE
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -507,7 +507,6 @@ services:
       build: ./elasticsearch
       volumes:
         - elasticsearch-data:/usr/share/elasticsearch/data
-        - elasticsearch-plugins:/usr/share/elasticsearch/plugins
       environment:
         - cluster.name=laradock-cluster
         - bootstrap.memory_lock=true
@@ -746,6 +745,4 @@ volumes:
   caddy:
     driver: "local"
   elasticsearch-data:
-    driver: "local"
-  elasticsearch-plugins:
     driver: "local"

--- a/elasticsearch/Dockerfile
+++ b/elasticsearch/Dockerfile
@@ -1,3 +1,3 @@
-FROM docker.elastic.co/elasticsearch/elasticsearch:5.4.1
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.2.3
 
 EXPOSE 9200 9300

